vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.kt

```diff
@@ -93,6 +96,11 @@ import java.util.*
 
 class VectorSettingsPreferencesFragment : PreferenceFragmentCompat(), SharedPreferences.OnSharedPreferenceChangeListener {
 
+    init {
+        // to support vector drawables below API 21 (till API 16)
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
```

Comment:
[The documentation](https://developer.android.com/reference/android/support/v7/app/AppCompatDelegate.html#setCompatVectorFromResourcesEnabled(boolean)) says this it take effect only for Activities created after this call. So I assume it's to late here.
Also the docs warns about updating Configuration after this call. And this is what we are doing when updating language or font size. So I will check if it is still working.
In my past experience, I already renounce to use vector drawable for application with min sdk level < 21.
So I would prefer to use good old png images, maybe those that already exist (for instance: unit_test_ko and unit_test_ok), rather than introducing vector here.

Reply:
No if there is configuration change it's not good practice to use this . I would use pngs then . 
Would make required changes .